target/linux/ath79/image/tiny-tp-link.mk

```diff
@@ -66,6 +66,16 @@ define Device/tplink_tl-mr3420-v2
 endef
 TARGET_DEVICES += tplink_tl-mr3420-v2
 
+define Device/tplink_tl-wa830re-v1
+  $(Device/tplink-4m)
+  SOC := ar7240
+  DEVICE_MODEL := TL-WA830RE
+  DEVICE_VARIANT := v1
+  TPLINK_HWID := 0x08300010
+  SUPPORTED_DEVICES += tl-wa830re-v1
```

Comment:
I don't think that's correct. However, it's hard to find the correct one from code, so easiest would be if you installed ar71xx and looked what's in /tmp/sysinfo/board_name

Reply:
Changed it, now sysupgrade doesn't complain.